### ISSUE TITLE
GraphQL Page API - Implement 'layout' field

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#19136]
Query: 
```
{
  page(url:"/destinations/costa-rica", pageMode: "live", languageId: "1") {
     layout {
      footer
      width
      header
      title
      body {
        rows {
          styleClass
          columns {
            preview
            width
            widthPercent
            left
            leftOffset
            styleClass
            containers {
              identifier
              uuid
            } 
          }
        } 
      }
      sidebar {
        preview
        width
        widthPercent
        location
        containers {
            identifier
            uuid
          } 
      }
    }
  }
}
```

Response:
```
{
    "data": {
        "page": {
            "layout": {
                "footer": "true",
                "width": null,
                "header": "true",
                "title": "anonymouslayout1598564677000",
                "body": {
                    "rows": [
                        {
                            "styleClass": null,
                            "columns": [
                                {
                                    "preview": false,
                                    "width": 12,
                                    "widthPercent": 100,
                                    "left": 0,
                                    "leftOffset": 1,
                                    "styleClass": null,
                                    "containers": [
                                        {
                                            "identifier": "5a07f889-4536-4956-aa6e-e7967969ec3f",
                                            "uuid": "2"
                                        }
                                    ]
                                }
                            ]
                        },
                        {
                            "styleClass": null,
                            "columns": [
                                {
                                    "preview": false,
                                    "width": 6,
                                    "widthPercent": 50,
                                    "left": 0,
                                    "leftOffset": 1,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "2"
                                        }
                                    ]
                                },
                                {
                                    "preview": false,
                                    "width": 6,
                                    "widthPercent": 50,
                                    "left": 6,
                                    "leftOffset": 7,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "3"
                                        }
                                    ]
                                }
                            ]
                        },
                        {
                            "styleClass": null,
                            "columns": [
                                {
                                    "preview": false,
                                    "width": 12,
                                    "widthPercent": 100,
                                    "left": 0,
                                    "leftOffset": 1,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "4"
                                        }
                                    ]
                                }
                            ]
                        },
                        {
                            "styleClass": null,
                            "columns": [
                                {
                                    "preview": false,
                                    "width": 10,
                                    "widthPercent": 83,
                                    "left": 1,
                                    "leftOffset": 2,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "5"
                                        }
                                    ]
                                }
                            ]
                        },
                        {
                            "styleClass": null,
                            "columns": [
                                {
                                    "preview": false,
                                    "width": 6,
                                    "widthPercent": 50,
                                    "left": 0,
                                    "leftOffset": 1,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "6"
                                        }
                                    ]
                                },
                                {
                                    "preview": false,
                                    "width": 6,
                                    "widthPercent": 50,
                                    "left": 6,
                                    "leftOffset": 7,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "7"
                                        }
                                    ]
                                }
                            ]
                        },
                        {
                            "styleClass": null,
                            "columns": [
                                {
                                    "preview": false,
                                    "width": 5,
                                    "widthPercent": 41,
                                    "left": 0,
                                    "leftOffset": 1,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "8"
                                        }
                                    ]
                                },
                                {
                                    "preview": false,
                                    "width": 6,
                                    "widthPercent": 50,
                                    "left": 6,
                                    "leftOffset": 7,
                                    "styleClass": "",
                                    "containers": [
                                        {
                                            "identifier": "//demo.dotcms.com/application/containers/default/",
                                            "uuid": "9"
                                        }
                                    ]
                                }
                            ]
                        }
                    ]
                },
                "sidebar": {
                    "preview": false,
                    "width": "small",
                    "widthPercent": 20,
                    "location": "left",
                    "containers": [
                        {
                            "identifier": "//demo.dotcms.com/application/containers/default/",
                            "uuid": "1"
                        },
                        {
                            "identifier": "5a07f889-4536-4956-aa6e-e7967969ec3f",
                            "uuid": "1"
                        }
                    ]
                }
            }
        }
    }
}
```

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19184

**Comment by dsilvam** [comment]
Note to QA: Postman test to run is `GraphQL.PageAPI.GivenRequestByURI_LayoutFieldContainsAllFields`

**Comment by victoralfaro-dotcms** [comment]
Working as expected
```
{
  "data": {
    "page": {
      "layout": {
        "footer": true,
        "width": null,
        "header": true,
        "title": "anonymouslayout1598564677000",
        "body": {
          "rows": [
            {
              "styleClass": null,
              "columns": [
                {
                  "preview": false,
                  "width": 12,
                  "widthPercent": 100,
                  "left": 0,
                  "leftOffset": 1,
                  "styleClass": null,
                  "containers": [
                    {
                      "identifier": "5a07f889-4536-4956-aa6e-e7967969ec3f",
                      "uuid": "2"
                    }
                  ]
                }
              ]
            },
            {
              "styleClass": null,
              "columns": [
                {
                  "preview": false,
                  "width": 6,
                  "widthPercent": 50,
                  "left": 0,
                  "leftOffset": 1,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "2"
                    }
                  ]
                },
                {
                  "preview": false,
                  "width": 6,
                  "widthPercent": 50,
                  "left": 6,
                  "leftOffset": 7,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "3"
                    }
                  ]
                }
              ]
            },
            {
              "styleClass": null,
              "columns": [
                {
                  "preview": false,
                  "width": 12,
                  "widthPercent": 100,
                  "left": 0,
                  "leftOffset": 1,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "4"
                    }
                  ]
                }
              ]
            },
            {
              "styleClass": null,
              "columns": [
                {
                  "preview": false,
                  "width": 10,
                  "widthPercent": 83,
                  "left": 1,
                  "leftOffset": 2,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "5"
                    }
                  ]
                }
              ]
            },
            {
              "styleClass": null,
              "columns": [
                {
                  "preview": false,
                  "width": 6,
                  "widthPercent": 50,
                  "left": 0,
                  "leftOffset": 1,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "6"
                    }
                  ]
                },
                {
                  "preview": false,
                  "width": 6,
                  "widthPercent": 50,
                  "left": 6,
                  "leftOffset": 7,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "7"
                    }
                  ]
                }
              ]
            },
            {
              "styleClass": null,
              "columns": [
                {
                  "preview": false,
                  "width": 5,
                  "widthPercent": 41,
                  "left": 0,
                  "leftOffset": 1,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "8"
                    }
                  ]
                },
                {
                  "preview": false,
                  "width": 6,
                  "widthPercent": 50,
                  "left": 6,
                  "leftOffset": 7,
                  "styleClass": "",
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/",
                      "uuid": "9"
                    }
                  ]
                }
              ]
            }
          ]
        },
        "sidebar": {
          "preview": false,
          "width": "small",
          "widthPercent": 20,
          "location": "left",
          "containers": [
            {
              "identifier": "//demo.dotcms.com/application/containers/default/",
              "uuid": "1"
            },
            {
              "identifier": "5a07f889-4536-4956-aa6e-e7967969ec3f",
              "uuid": "1"
            }
          ]
        }
      }
    }
  }
}
```

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.8 // Postgres // FF
